orbotservice/build.gradle

```diff
@@ -41,11 +41,13 @@ android {
         lintConfig file("../lint.xml")
     }
 
+    packagingOptions {
+        exclude 'META-INF/androidx.localbroadcastmanager_localbroadcastmanager.version'
```

Comment:
and this

Reply:
last I checked it was required because there was a conflict